### ISSUE TITLE
Unable to reach the server using a self-signed SSL certificate

### ISSUE THREAD

**Issue by alchy58** [yDelouis/selfoss-android#48]
Hi !

Sorry for my bad english !
I'm using selfoss since many time ! I've just discovered the Android app so I tried to test it.

The app give me the error message "Unable to reach the server" when I'm using a self-signed SSL certificate. When I'm switching to HTTP, my feeds appear !

Thank you for your work dude !

**Comment by yDelouis** [comment]
If the problem is the certificate, a dialog should appear to ask you if you want to allow any certificate.
Are you sure, the https url is reachable from your phone (try in a browser) ?

**Comment by alchy58** [comment]
Yes it's rechable. I use Firefox to browse directly the website.
I confirm that I don't have any dialog asking me to allow hte certificate.

Sony Xperia Z1 Compact
Android 4.4.2

Which infos do you need ?

**Comment by Spy-Seth** [comment]
I've a issue to connect with my selfoss instance:

![screenshot_2015-08-01-12-30-44](https://cloud.githubusercontent.com/assets/95120/9021584/f22ded7a-3849-11e5-93ac-ea202bc06d31.png)

I've a self-signed certificate too, but the application ask me to accept it. But I can't connect.

I use selfoss 1.13. I can provide you a login/pass to test on my instance if this can help.

**Comment by Wonderfall** [comment]
Same here, yet I'm using a certificate issued by StartSSL.

**Comment by phxql** [comment]
If you're using a self signed certificate, make sure that this certificate has an AlternativeName on it, see [here](https://stackoverflow.com/questions/21488845/how-can-i-generate-a-self-signed-certificate-with-subjectaltname-using-openssl). I just searched 2 hours for the cause, because the app just said "Unable to reach the server". Now it works.
